suite-native/accounts/src/components/AddAccountsButton.tsx

```diff
@@ -23,13 +26,16 @@ export const AddAccountButton = () => {
         });
     };
 
-    const navigateToAddCoinAccount = () =>
-        navigation.navigate(RootStackRoutes.AddCoinAccountStack, {
-            screen: AddCoinAccountStackRoutes.AddCoinAccount,
-            params: {
-                flowType: 'accounts',
-            },
-        });
+    const navigateToAddCoinAccount = () => {
+        if (isDeviceConnectEnabled && isAddCoinAccountEnabled) {
```

Comment:
Why is here check for `isDeviceConnectEnabled`? Should not be `isAddCoinAccountEnabled ` enough?

Reply:
Since these are independent feature flags this makes sure, that even if we make `isAddCoinAccountEnabled = true` and `isDeviceConnectEnabled = false` by mistake, everything works as intended. 

End game is to remove the Feature flag with proper release and keep only `isDeviceConnectEnabled`. Because then the `isAddCoinAccountEnabled` wont be needed.

Reply:
d7ea06eeea